src/views/GenKeyfile.js

```diff
@@ -101,6 +107,17 @@ class GenKeyfile extends React.Component {
         </Button>
       : ''
 
+    let networkSeedWarning
+
+    if (this.state.networkSeed === '') {
+      networkSeedWarning = (
+        <P>
+          <b>Warning: </b>
+          { "No network seed detected. To generate a keyfile, you must reset your networking keys." }
```

Comment:
How about "Try logging in with your master ticket or management mneumonic if you want to keep your current networking keys" ?

Reply:
Problem is it would only work if the current keys were actually set using those.  
Maybe "Try logging in with your master ticket or management mnemonic. Alternatively, reset your keys."? All of this feels pretty aggressive if the user isn't aware that this is "quick and easy".